zkevm-circuits/src/witness/bytecode.rs

```diff
@@ -24,16 +27,14 @@ impl Bytecode {
     /// Assignments for bytecode table
     pub fn table_assignments<F: Field>(
         &self,
-        challenges: &Challenges<Value<F>>,
-    ) -> Vec<[Value<F>; 5]> {
+        _challenges: &Challenges<Value<F>>,
```

Comment:
This table_assignment is not a trait method. I think the challenge can be safely removed.
```suggestion
```